.teamcity/src/subprojects/release/publishing/ProjectPublishing.kt

```diff
@@ -0,0 +1,28 @@
+package subprojects.release.publishing
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import subprojects.*
+import subprojects.build.core.*
+
+data class Build(val name: String, val build: BuildType?)
+
+object ProjectPublishing : Project({
+    id("ProjectPublishing")
+    name = "Publishing"
+    description = "Publish artifacts to repositories"
+
+    val publishingTargets = listOf(
+        Build("JVM", generatedBuilds["${linux.name}${java11.name}"]),
+        Build("JavaScript", generatedBuilds[js.name]),
+        Build("Windows", generatedBuilds[windows.name]),
```

Comment:
This is complicated because it's not clear what should I pass as the key to `generatedBuilds`. For example, what is `linux.name`; why not `agentString`? The line `generatedBuilds["${linux.name}${java11.name}"])` looks like magic because I need to know how and what to glue.

Reply:
Will be addressed later on